samples/GraphQL.Federation.SchemaFirst.Sample1/MyFederatedResolver.cs

```diff
@@ -20,10 +20,10 @@ public MyFederatedResolver(Func<Data, int, Task<T?>> resolver)
 
     public bool MatchKeys(IDictionary<string, object?> representation) => true;
 
-    public object ParseRepresentation(IObjectGraphType graphType, IDictionary<string, object?> representation)
+    public object ParseRepresentation(IComplexGraphType graphType, IDictionary<string, object?> representation)
         => (int)Convert.ChangeType(representation["id"], typeof(int), CultureInfo.InvariantCulture)!;
 
-    public async ValueTask<object?> ResolveAsync(IResolveFieldContext context, IObjectGraphType graphType, object parsedRepresentation)
+    public async ValueTask<object?> ResolveAsync(IResolveFieldContext context, IComplexGraphType graphType, object parsedRepresentation)
```

Comment:
If any changes were needed to an application, it would be approximately this complex.